### PR TITLE
[WIP]: Phase overlap tool

### PR DESCRIPTION
**Pull request by catherine-martlin**
These are the changes to get the phase overlap tool up and running on the website.

### CODE REVIEW

File: exoctk/phase_constraint_overlap/phase_constraint_overlap.py [hunk 1]
```diff
@@ -32,6 +31,16 @@
 
 from exoctk.utils import get_target_data
 
+def build_target_url(target_name):
```
Comment (mfixstsci) [inline exoctk/phase_constraint_overlap/phase_constraint_overlap.py]: Is this supposed to be here?

File: exoctk/phase_constraint_overlap/phase_constraint_overlap.py [hunk 2]
```diff
@@ -51,12 +60,13 @@ def calculate_phase(period, obsDur, winSize):
         maxphase : float
             The maximum phase constraint. '''
 
-    minphase = 1.0 - ((obsDur + winSize)/2.0/24/period)
-    maxphase = 1.0 - ((obsDur - winSize)/2.0/24/period)
+    minphase = 1.0 - ((0.5 + obsDur + winSize)/2.0/24/period)
```
Comment (mfixstsci) [inline exoctk/phase_constraint_overlap/phase_constraint_overlap.py]: Where does the `0.5` come from? Is it from the literature?

File: exoctk/phase_constraint_overlap/phase_constraint_overlap.py [hunk 3]
```diff
@@ -67,14 +77,36 @@ def calculate_obsDur(transitDur):
...
-            The duration of the observation in hours. '''
+            The duration of the observation in hours. Maximum of 6 hours. '''
 
     obsDur = np.min((6, 3*transitDur+1))
 
     return obsDur
 
 
-def phase_overlap_constraint(target_name, period=None, t0=None, obs_duration=None, window_size=None):
+def get_transit_details(target_name):
```
Comment (mfixstsci) [inline exoctk/phase_constraint_overlap/phase_constraint_overlap.py]: We have this portion of source code taken care of now. Feel free to remove this function.

File: exoctk/phase_constraint_overlap/phase_constraint_overlap.py [hunk 4]
```diff
@@ -98,35 +128,42 @@ def phase_overlap_constraint(target_name, period=None, t0=None, obs_duration=Non
...
+    if window_size == None: 
+        window_size = 1.0
 
     minphase, maxphase = calculate_phase(period, obs_duration, window_size)
+
+    return minphase, maxphase
     
-    # Is this the return that we want? Do we need to use t0 for something? 
-    print('MINIMUM PHASE: {}, MAXIMUM PHASE: {}'.format(minphase, maxphase))
+    # Is this the return that we want?
```
Comment (mfixstsci) [inline exoctk/phase_constraint_overlap/phase_constraint_overlap.py]: Thats fine, we can just delete the code and take care of this later.

File: exoctk/phase_constraint_overlap/phase_constraint_overlap.py [hunk 5]
```diff
@@ -98,35 +128,42 @@ def phase_overlap_constraint(target_name, period=None, t0=None, obs_duration=Non
...
+    #print('MINIMUM PHASE: {}, MAXIMUM PHASE: {}'.format(minphase, maxphase))
 
 # Need to make entry point for this!
-if __name__ == '__main__':
-    args = docopt(__doc__, version='0.1')
+#if __name__ == '__main__':
```
Comment (mfixstsci) [inline exoctk/phase_constraint_overlap/phase_constraint_overlap.py]: We can just delete the code and take care of this later.

**Comment by catherine-martlin** [comment]
This PR covers a lot of the same things that @mfixstsci's PR #313 covers so I'm closing this and allowing that to handle this functionality so we don't have weird merge conflicts.

**Pull request closed without merging**